Formula/tideways-cli.rb

```diff
@@ -2,7 +2,7 @@
 
 class TidewaysCli < Formula
     homepage 'https://tideways.com'
-    version '1.1.0'
+    version "1.1.0
```

Comment:
Closing double quote missing?  Same in tideways-daemon.rb

Reply:
Good catch, fixed.